update_ombi.sh

```diff
@@ -170,6 +173,10 @@ if [ -z "${port}" ]; then
     .log 5 "Port not parsed...setting to default: $defaultport"
     port="$defaultport"
 fi
+if [ -z "${apikey}" ]; then
```

Comment:
This is just here to match the other code that parses values. I'm not sure where the API key could be parsed from so if it's not set in the default variables it won't be used (per the code below).